src/qml/ChatPage.qml

```diff
@@ -59,18 +59,23 @@ Kirigami.Page {
 				Rectangle {
 					id: avatar
 
-					width: height
-					height: parent.height
+					width: 40
+					height: 40
+					radius: width*0.5
```

Comment:
The radius? I thought it fits there, because it has something to do with styling and size too. I will correct the *.